arm-notificationhubs/2016-03-01/swagger/notificationhubs.json

```diff
@@ -129,6 +129,52 @@
           }
         }
       },
+	  "patch": {
+		"tags": [
+          "Namespaces"
+        ],
+		"operationId": "Namespaces_Patch",
+		"description": "Patches the existing namespace",
+		"parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
```

Comment:
@amarzavery Yesterday you were showing me to put something here, is this applicable to this PR as well.